### DISCUSSION TITLE
TextElementGroupState updateElements

### DISCUSSION THREAD

**Discussion by blueRaining** [heremaps/harp.gl#2304] [Q&A]
`	updateElements(filter: TextElementFilter) {
		for (const elementState of this.m_textElementStates) {
			const textDistance = filter(elementState);
			elementState.update(textDistance);
		}
	}`
how it to filter the elementState?when the textDistance is undefined,should we delete it from m_textElementStates?

**Comment by nzjony** [comment]
@atomicsulfate , do you know?
